### PR TITLE
Drop Debian < 8 and Ubuntu < 14.04 code

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Review by bastelfreak — APPROVED** [review]

**Comment by ekohl** [comment]
Since this is breaking, I'm interesting in @puppetlabs/modules weighing in. Sadly, I can't ping them.

**Comment by sheenaajay** [comment]
@ekohl Thanks for the PR. Its important to remove stale code to keep the maintenance easier. Was talking with the team and looks like its not supported. https://puppet.com/docs/pe/2019.8/supported_operating_systems.html#supported_operating_systems
But it will definitely break existing users. We need to inform community in advance about the release of major version and its impact. Thank you.

**Comment by sheenaajay** [comment]
@ekohl Will  be adding  the details about the PR in our this week blog post to inform the community for the major release of this module and its changes.

**Review by sheenaajay — APPROVED** [review]

File: manifests/mod/suphp.pp [hunk 1]
```diff
@@ -6,7 +6,7 @@
 class apache::mod::suphp (
 ) {
   if  ($facts['os']['name'] == 'Ubuntu' and versioncmp($facts['os']['release']['major'], '15.10') >= 0) or
-  ($facts['os']['name'] == 'Debian' and versioncmp($::operatingsystemrelease, '8') >= 0) {
+  $facts['os']['family'] == 'Debian' {
```
Comment (ekohl) [inline manifests/mod/suphp.pp:9]: I now reread this and this looks like a bad rebase from my side. This also drops Ubuntu < 15.10 support here.

Comment (ekohl) [reply manifests/mod/suphp.pp:9]: https://github.com/puppetlabs/puppetlabs-apache/pull/2193 should fix that.

Comment (sheenaajay) [reply manifests/mod/suphp.pp:9]: Thank you @ekohl for the fixes.

**Pull request merged**